.bazelrc

```diff
@@ -36,3 +36,5 @@ build:asan --copt -fno-omit-frame-pointer
 build:asan --linkopt -fsanitize=address
 
 build:create_static_linked_executables --linkopt=-fuse-ld=bfd --features=-supports_start_end_lib --//bazel:create_static_linked_executables
+
+try-import %workspace%/.user.bazelrc
```

Comment:
Can you add a comment, something like
```
# User-chosen additional development configuration
```

I'd probaly make it a visible file without prefix-dot: `user.bazelrc`

If you add it to the `.gitignore` it does not accidentally get submitted

Reply:
Looks like bazel itself agrees with a visible `user.bazelrc` : https://github.com/bazelbuild/bazel/blob/master/.bazelrc#L61

Reply:
Ah, also found a documentation with that suggestion https://bazel.build/configure/best-practices#bazelrc-file